src/commands/paymaster/addTarget.ts

```diff
@@ -0,0 +1,46 @@
+import { Argv } from "yargs";
+import { error, info, success } from "signale";
+import { getLogger } from "../../logger";
+import { addTargetToPaymaster } from "../../implementations/paymaster/addTarget";
+import { PaymasterAddTargetCommand } from "./paymaster-command.type";
+import { withGasPriceOption, withNetworkAndKeyOption } from "../shared";
+import { getEtherscanAddress } from "../../utils";
+
+const { trace } = getLogger("paymaster:add-target");
+
+export const command = "add-target [options]";
+
+export const describe = "Registers a contract with a paymaster deployed on the blockchain";
+
+export const builder = (yargs: Argv): Argv =>
+  withGasPriceOption(
+    withNetworkAndKeyOption(
+      yargs
+        .option("targetAddress", {
```

Comment:
I think we usually use kebab-case here as well, could check to confirm and change if that's the case ?

Reply:
I check, dont think it matters but i will change it

Reply:
There seems to be a few standards
https://github.com/Open-Attestation/open-attestation-cli/blob/master/src/commands/dns/txt-record/create.ts#L23

Reply:
https://github.com/Open-Attestation/open-attestation-cli/blob/master/src/commands/sign.ts#L13

Reply:
I see more kebab-case, can fix yours to follow the convention ? :)